monthly-contributions/src/main/scala/com/gu/support/workers/lambdas/UpdateMembersDataAPI.scala

```diff
@@ -14,6 +17,10 @@ class UpdateMembersDataAPI(servicesProvider: ServiceProvider = ServiceProvider)
   def this() = this(ServiceProvider)
 
   override protected def servicesHandler(state: UpdateMembersDataAPIState, context: Context, services: Services): Future[Unit] = {
-    services.membersDataService.update(state.user.id, state.user.isTestUser).map(_ => Unit)
+    services.membersDataService.update(state.user.id, state.user.isTestUser).map(_ => putCloudWatchMetrics)
```

Comment:
The `Future` that this returns will complete before `putCloudWatchMetrics` is complete, so the lambda may terminate before the cloudwatch request has happened.  If you modify putCloudWatchMetrics to return a `Future` you can use `flatMap` instead of `map` which will fix this problem